### PR TITLE
Create LICENSE

### PR DESCRIPTION
**Pull request by BenLloydPearson**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2017 Oath
```
Comment (RomanTysiachnik) [inline LICENSE]: Due to that [example](https://git.ouroath.com/Open-Source/Oath-Open-Source-Skeleton/blob/master/LICENSE-MIT), there shouldn't be a copyright mark.
Please replace on `Copyright 2018 Oath Inc.`

Comment (BenLloydPearson) [reply LICENSE]: The first commit to this project was in 2017 so the copyright year should reflect this.

Comment (MarkGerl) [reply LICENSE]: @RomanTysiachnik make this 'Copyright (c) 2017 Oath Inc.' ... don't forget the "Inc."  Thanks!

**Comment by BenLloydPearson** [comment]
Why was this closed?

File: LICENSE (hunk 1 above)
Comment (BenLloydPearson) [reply LICENSE]: 'Inc.' has been added.

**Comment by MarkGerl** [comment]
@BenLloydPearson our CI system is down; can you force this PR through and merge it please.

**Pull request merged**